py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2708,6 +2708,23 @@ def aten_ops_scalar_tensor(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.log2.default)
+def log2(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.unary.log2(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+    )
```

Comment:
Should this be `log10` for this PR?

Reply:
yeah looks like I was referring to the wrong PR when I was rebase onto master branch just now. Thanks